IE11 Support - Note and sample fixes (#146)
OfficeDev/office-scripts-docs: docs/overview/code-editor-environment.md

Office Scripts do not support the usage of external, third-party JavaScript libraries. You are currently unable to call any library other than the Office Scripts APIs from a script. You do still have access to any [built-in JavaScript object](../develop/javascript-objects.md), such as [Math](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Math).

## Browser support

> Office Scripts work in any browser that [supports Office for the web](https://support.microsoft.com/office/ad1303e0-a318-47aa-b409-d3a5eb44e452). However, some JavaScript features aren't supported in Internet Explorer 11. Any features introduced in [ES6 or later](https://www.w3schools.com/Js/js_es6.asp) won't work with Internet Explorer 11. If you have people in your organization still using that browser, be sure to test your scripts in that environment when sharing them.

Reviewer: ```suggestion
Office Scripts work in any browser that [supports Office for the web](https://support.microsoft.com/office/ad1303e0-a318-47aa-b409-d3a5eb44e452). However, some JavaScript features aren't supported in Internet Explorer 11 (IE 11). Any features introduced in [ES6 or later](https://www.w3schools.com/Js/js_es6.asp) won't work with IE 11. If people in your organization still use that browser, be sure to test your scripts in that environment when sharing them.
```
